### PR TITLE
Hong hty

### PR DESCRIPTION
**Pull request by hoanghong0902**
Fix flyway issue

### CODE REVIEW

**Review by mainhuitvt96 — APPROVED** [review]

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@
 spring.flyway.url=jdbc:mysql://localhost:3306/aurora_shop
 spring.flyway.user=root
 spring.flyway.password=piitiamo1702
-spring.flyway.enabled=false
+spring.flyway.enabled=true
```
Comment (DUYNGOC97) [inline src/main/resources/application.properties]: change true to false

**Review by DUYNGOC97 — COMMENTED** [review]
change flyway flag to false

File: src/main/resources/application.properties (hunk 1 above)
Comment (hoanghong0902) [reply src/main/resources/application.properties]: resolved

**Review by DUYNGOC97 — APPROVED** [review]
LGTM

**Pull request merged**